dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/FriendlyFileTypeUtil.java

```diff
@@ -0,0 +1,51 @@
+package edu.harvard.iq.dataverse.common;
+
+import edu.harvard.iq.dataverse.common.files.mime.ShapefileMimeType;
+import edu.harvard.iq.dataverse.persistence.datafile.DataFile;
+
+import java.util.MissingResourceException;
+
+public class FriendlyFileTypeUtil {
+
+    public static String getUserFriendlyFileType(DataFile dataFile) {
+        String fileType = dataFile.getContentType();
+
+        if (fileType != null) {
```

Comment:
fileType cannot be null, Entity has @ NotBlank annotation.

Reply:
:+1: